examples/2D/denoising2D_RGB/01_training.ipynb

```diff
@@ -84,6 +86,47 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Here we just load a single image and inspect the dimensions.\n",
```

Comment:
Sorry for jumping in. I was curious and couldn't resist reading this in diagonal.

> Here we just load a single image and inspect the dimensions.

I'd remove fill words like 'just' here.
For people not familiar with `numpy` or `PIL.Image` this single `just` can be received in a demotivating fashion, i.e. for someone that doesn't understand `np.array(Image.open('data/longBeach.png'))` at plain sight is likely to feel incompetent by means of the implications behind `just`. ;) This feeling is something you don't want in the first notebook they read through when learning `n2v`.